proposals/authorization-ucr/index.bs

```diff
@@ -1694,7 +1694,7 @@ or limiting access to a [=resource=] or [=collection=] by an [=agent=].
   Related use cases: [[#collection-readcreatedelete]]
 </div>
 
-### The system shall not rely on the URI path to identity [=resources=] or [=collections=] ### {#req-uripath}
```

Comment:
The protocol should not rely on the URI structure to infer semantics about the type of resources.

Reply:
I would keep "the system" which refers to "the authorization system" for the sake of consistency and because the UCR is about the authorization system, not about the solid protocol.

It's a good thing that the protocol allows to do that. But this is not the requirement we're expressing here.

Reply:
I like your suggestion otherwise.

Reply:
@bblfish I ammended the text following your suggestion and keeping a consistent wording with the other UCRs.